js/stores/eventStore.js

```diff
@@ -101,6 +108,16 @@ const doAction = (action) => {
       addPageView(action.frameProps.get('location'))
       lastActiveTabId = action.frameProps.get('tabId')
       break
+    case AppConstants.APP_WINDOW_BLURRED:
+      windowBlurred(action.appWindowId)
+      break
+    case AppConstants.APP_IDLE_STATE_CHANGED:
+      if (action.idleState !== 'active') {
+        addPageView(null)
```

Comment:
it just logs the timestamp when the browser became inactive. We use the same thing for closing all windows